test/system_settings_test.rb

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SystemSettings"
+
+describe "Yast::SystemSettings" do
+  Yast.import "Bootloader"
+
+  subject(:settings) { Yast::SystemSettings }
+
+  before { settings.main }
+
+  describe "#GetPossibleElevatorValues" do
+    it "returns an array with possible schedulers" do
+      expect(settings.GetPossibleElevatorValues).to eq(["cfq", "noop", "deadline"])
```

Comment:
or https://www.relishapp.com/rspec/rspec-expectations/v/3-0/docs/built-in-matchers/contain-exactly-matcher
so contain-exactly is also possible